Tasks/AzureRmWebAppDeployment/task.json

```diff
@@ -31,6 +31,12 @@
             "name": "AdditionalDeploymentOptions",
             "displayName": "Additional Deployment Options",
             "isExpanded": false,
+            "visibleRule": "WebAppKind != \"\""
+        },
+        {
+            "name": "PostDeploymentOptions",
```

Comment:
I will change this name based on Atul's input.

Reply:
I think you should do it other way round; The script should be in Post Deployment options. And the Other options in "Additional Deployment options"